sdk/ml/azure-ai-ml/tests/dsl/unittests/test_dsl_pipeline.py

```diff
@@ -2581,3 +2581,19 @@ def root_pipeline():
         pipeline_job.settings.default_compute = "cpu-cluster"
         validate_result = pipeline_job._validate()
         assert validate_result.error_messages == {}
+
+    def test_dsl_pipeline_component_early_available_property(self):
+        component_func = load_component("./tests/test_configs/components/streaming_component_basic.yml")
```

Comment:
what's the behavior if we are using a registered component in pipeline job yaml

Reply:
it should be same as anonymous, let me verify and add a test on this.

Reply:
I have created a component with early available output, and consume it in a pipeline job yaml.
the created pipeline job output has correct early available output as expected, however, as we do not load the component locally in such workflow, SDK will not resolve the arm id and it seems that we cannot check this property as a test.